cmd/readium/cmd/root.go

```diff
@@ -0,0 +1,26 @@
+package cmd
+
+import (
+	"os"
+
+	"github.com/spf13/cobra"
+)
+
+// rootCmd represents the base command when called without any subcommands
+var rootCmd = &cobra.Command{
+	Use:   "readium",
+	Short: "Readium Go toolkit",
+}
+
+// Execute adds all child commands to the root command and sets flags appropriately.
+// This is called by main.main(). It only needs to happen once to the rootCmd.
+func Execute() {
+	err := rootCmd.Execute()
+	if err != nil {
+		os.Exit(1)
```

Comment:
No need to log/print the error?

Reply:
Yes curious, it was generated by `cobra-cli`.

Reply:
Apparently the error is already printed by Cobra, this check is only to set the exit code.